lib/puppet/parser/functions/platform_fretta.rb

```diff
@@ -25,11 +25,13 @@ module Parser
     #
     module Functions
       newfunction(:platform_fretta, type: :rvalue) do |_args|
-        data = lookupvar('os')
+        data = lookupvar('cisco')
         return '' if data.nil?
 
-        pat = '7.0\(3\)F'
-        data['release']['full'].match(pat) ? true : false
+        data['inventory'].each do |_x, slot|
+          return true if slot['pid'][/-R/]
```

Comment:
In the `node_utils` function we check for both `FM-R` and `-R`.  Do we need both checks here or can we simplify it to just check `-R`?

Reply:
Changed NU to use -R only